.github/workflows/cargo-check.yml

```diff
@@ -19,15 +19,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Install latest rust
-        uses: actions-rs/toolchain@v1
+      - name: Use correct Rust toolchain
+        shell: bash
+        run: |
+          cp --update=none rust/rust-toolchain.toml ./ || true
+      - name: Install rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1
```

Comment:
Oh nice. Had we used this originally, @iterion 's suggestion would've worked - what made you search this out?

Reply:
`actions-rs/toolchain` was so ancient that it didn't support the TOML formatted `rust-toolchain` file https://github.com/actions-rs/toolchain/issues/126. Also, I guess it was technically expanding scope, but since I was mucking in the options, I figured it was time to migrate off of something that was unmaintained for so long.